StRoot/RTS/src/DAQ_TPC23/itpc23.cxx

```diff
@@ -718,7 +719,7 @@ u_int *itpc23::fee_non_trgd(u_int *start)
 	int fee_words = 0 ;
```

Comment:
Jeff and I have noted to Tonko that this local `fee_words` should perhaps not be local, but we believe this is not worth holding up this commit as even without any change to this line, this commit is an important step forward.